cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -390,6 +391,7 @@ public void execute() {
                 // create the jar.
                 .addTask(new CreateJarTask(this.dumpBIR, skipCopyLibsFromDist))
                 .addTask(new CopyResourcesTask(), isSingleFileBuild)
+                .addTask(new CopyObservabilitySymbolsTask(), isSingleFileBuild)
```

Comment:
I think we don't need this for the test. But not sure, we need this for run command. @a5anka Are we trying to observe using run common?

Reply:
Actually, I added this to run command. In the future users might need to run with observability features fro advanced debugging use cases with Choreo. Therefore it will be easier for the user if we have this enabled for run command. Isn't it?